test/schema/me/lot_standing.js

```diff
@@ -179,4 +179,112 @@ describe("LotStanding type", () => {
       })
     })
   })
+
+  it("correctly determines if a lot is part of a live auction", () => {
+    gravity
```

Comment:
The tests no longer need to use a rewired `gravity` when using the right data loaders. Here’s an example of how you’d inject stubbed data loaders: https://github.com/artsy/metaphysics/blob/37e600b2f7ff97b08b84a6291cbdbe2e7bd0c181/test/schema/me/conversation/send_message_mutation.js#L33-L51